### PR TITLE
Implement IsZero method

### PR DESCRIPTION
**Pull request by emilgpa**

### CODE REVIEW

File: uuid.go [hunk 1]
```diff
@@ -307,6 +307,11 @@ func (u *UUID) UnmarshalBinary(data []byte) (err error) {
 	return
 }
 
+// IsZero returns true if UUID is zero
```
Comment (shurcooL) [inline uuid.go]: Nitpick. This comment's sentence doesn't end with a period. Other comments do, so this is slightly inconsistent.

**Comment by shurcooL** [comment]
I left a style suggestions comment, but I have no comment about the change at a higher level (I'm not closely familiar with the project's goals).

**Comment by dashaus** [comment]
suggestions done!

**Comment by satori** [comment]
I would question what is an actual value of adding this method instead of using `u == uuid.Nil`.

**Comment by dashaus** [comment]
It is much more idiomatic, less verbose and more comfortable.

**Comment by YuriFotiNesquik** [comment]
> It is much more idiomatic, less verbose and more comfortable.

Is that an advertisement for a new Iphone?

**Comment by kwk** [comment]
I just realized that I've added a PR that supersedes this one without knowing #43. @satori can you please have a look?

**Comment by satori** [comment]
Closing as this method doesn't produce much value for a library in my opinion.
I would like to keep implementation as simple as possible.

**Pull request closed without merging**